src/main/java/com/questhelper/runeliteobjects/extendedruneliteobjects/RuneliteObjectManager.java

```diff
@@ -481,13 +496,9 @@ private void addReplacedNpcOptions(ExtendedRuneliteObject extendedRuneliteObject
 		}
 	}
 
-	public void setActive(ExtendedRuneliteObject extendedRuneliteObject)
+	private void setVisible(ExtendedRuneliteObject extendedRuneliteObject, boolean visible)
```

Comment:
nit: This could be inlined where it's used, unless there's value in being able to breakpoint here instead of in `ExtendedRuneliteObject`'s `setVisible` function